internal/pkg/template/templates/workloads/partials/cf/nlb.yml

```diff
@@ -146,6 +146,8 @@ NLBCustomDomainFunction:
     Runtime: nodejs12.x
 
 NLBCustomDomainRole:
+  Metadata:
+    'aws:copilot:description': "An IAM role for NLBCustomDomainAction"
```

Comment:
I feel like we should be explaining what they're for rather than saying the logical ID that uses this IAM role. 

```suggestion
    'aws:copilot:description': "An IAM role to update the environment route53 hosted zone"
```